tracee-ebpf/tracee/events_pipeline.go

```diff
@@ -134,7 +124,7 @@ func (t *Tracee) decodeEvents(done <-chan struct{}) (<-chan *external.Event, <-c
 				PIDNS:               int(ctx.PidID),
 				ProcessName:         string(bytes.TrimRight(ctx.Comm[:], "\x00")),
 				HostName:            string(bytes.TrimRight(ctx.UtsName[:], "\x00")),
-				ContainerID:         containerId,
+				ContainerID:         t.containers.GetCgroupInfo(ctx.CgroupID).ContainerId,
```

Comment:
for each received event, you're now calling GetCgroupInfo() and, even if cgroupInfo exists (most of the times), it still requires a RO mutex lock (as fyio, I'm not sure about performance hit here, when considering tons of events).